ui/apps/platform/src/services/ReportsService.ts

```diff
@@ -144,7 +144,15 @@ export function fetchViewBasedReportHistory({
     sortOption,
     showMyHistory,
 }: FetchViewBasedReportHistoryServiceParams): Promise<ViewBasedReportSnapshot[]> {
-    const params = getListQueryParams({ searchFilter, sortOption, page, perPage });
+    const params = queryString.stringify(
+        {
+            reportParamQuery: {
+                query: getRequestQueryStringForSearchFilter(searchFilter),
+                pagination: getPaginationParams({ page, perPage, sortOption }),
+            },
+        },
+        { arrayFormat: 'repeat', allowDots: true }
+    );
```

Comment:
I thought we must have had a general function that encapsulates these `qs` options, but it doesn't look like we do. 🫤

Reply:
The `getListQueryParams` would have worked nicely for us, but it turns out that the API requires `reportParamQuery` at the top-level. So I made it similar to `fetchReportHistory` and `fetchComplianceReportHistory`. I should have expected the same structure. Definitely could make a reporting-specific util function to use in all three locations